lib/modules/datasource/index.spec.ts

```diff
@@ -43,6 +43,24 @@ class DummyDatasource extends Datasource {
   }
 }
 
+class DummyDatasource2 extends Datasource {
+  override readonly customRegistrySupport = false;
+  override defaultRegistryUrls = () => ['https://reg1.com'];
```

Comment:
```suggestion
  override get defaultRegistryUrls() { return ['https://reg1.com']; }
```

i don't think we need these new tests which my getter suggestions